lib/lita/adapters/slack/slack_user.rb

```diff
@@ -11,13 +11,13 @@ def from_data(user_data)
               user_data['id'],
               user_data['name'],
               user_data['real_name'],
-              user_data
+              user_data['profile']
             )
           end
 
           # @api private
           def from_data_array(users_data)
-            users_data.map { |user_data| from_data(user_data) }
+            users_data["members"].map { |user_data| from_data(user_data) }
```

Comment:
Data from the `users.list` endpoint now comes wrapped in a `members` array ([source](https://api.slack.com/methods/users.list#responses))